src/main/java/io/lettuce/core/cluster/pubsub/RedisClusterPubSubListener.java

```diff
@@ -67,4 +67,16 @@ public interface RedisClusterPubSubListener<K, V> {
      */
     void punsubscribed(RedisClusterNode node, K pattern, long count);
 
+    /**
+     * Subscribed to a shard channel.
+     *
+     * @param node the {@link RedisClusterNode} from which the {@code message} originates.
+     * @param shardChannel Shard channel
+     * @param count Subscription count.
+     */
+    default void ssubscribed(RedisClusterNode node, K shardChannel, long count){
+        throw new UnsupportedOperationException(
+                "This operation is not available by default! Use one of available pub/sub adapters or override this method in your class.");
```

Comment:
May I suggest that we rephrase this with a specific instruction? E.g.
"No default implementation. Please provide one in your custom listener."

Reply:
i ll rephrase it with next iterations